services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs.py

```diff
@@ -303,15 +306,29 @@ async def start_job(
     job_name = _compose_job_resource_name(solver_key, version, job_id)
     _logger.debug("Start Job '%s'", job_name)
 
-    task = await director2_api.start_computation(
-        project_id=job_id,
-        user_id=user_id,
-        product_name=product_name,
-        cluster_id=cluster_id,
-        groups_extra_properties_repository=groups_extra_properties_repository,
-    )
-    job_status: JobStatus = create_jobstatus_from_task(task)
-    return job_status
+    if pricing_spec := JobPricingSpecification.create_from_headers(request.headers):
```

Comment:
What is the unit?

Reply:
Good question! I have to admit I don't entirely understand the reason why there is a `plan` and a `unit`. But the point is that a pricing plan can have multiple units. Looking at the endpoints Matus implemented I believe a plan is associated with a service/product and the unit is closer to the individual nodes. So I think the use case is that one is running several nodes in osparc with the same service, then one can set different units which will probably determine the hardware (I guess). Please correct me here if I am wrong @matusdrobuliak66.

A pricing unit is given by the following data:
```
{
  "pricingUnitId": 1,
  "unitName": "string",
  "unitExtraInfo": {},
  "currentCostPerUnit": 0,
  "default": true
}
```
and a pricing plan is given by the following data:
```
{
  "pricingPlanId": 1,
  "displayName": "string",
  "description": "string",
  "classification": "TIER",
  "createdAt": "2023-10-20T09:13:31.011Z",
  "pricingPlanKey": "string",
  "pricingUnits": [
    {
      "pricingUnitId": 1,
      "unitName": "string",
      "unitExtraInfo": {},
      "currentCostPerUnit": 0,
      "default": true
    }
  ]
}
```